app/main.py

```diff
@@ -2,0 +1,41 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
+    path = ""
+
+    try:
+        if not isinstance(command, str):
+            raise TypeError("Command must be string")
+        if command[:2] != "mv":
+            raise ValueError("Command must start with keyword 'mv'")
+        if len(command_list) != 3:
+            raise ValueError(
+                "Command must include 'mv', source file name and new file name"
+            )
+    except (TypeError, ValueError) as e:
+        print(e)
+        return
+
+    if "/" in command_list[2]:
+        path_list = command_list[2].split("/")[:-1]
+
+        for directory in path_list:
+            path += os.path.join(directory + "/")
```

Comment:
Use `os.path.join` to concatenate directory paths instead of manually adding slashes. This will make your code more robust and cross-platform.